Fixed #35344 -- Corrected output_field of resolved columns of table alias for GeneratedFields. (#18034)
django/django: docs/releases/5.0.4.txt

* Fixed a bug in Django 5.0 that caused a migration crash on PostgreSQL 15+
  when adding a partial ``UniqueConstraint`` with ``nulls_distinct``
  (:ticket:`35329`).

> * Fixed a bug in Django 5.0 that caused a crash of some queries where a table
>   is joined with an alias and a lookup on a ``GeneratedField`` of the aliased
>   table is performed (:ticket:`35344`).

Reviewer: ```suggestion
* Fixed a crash in Django 5.0 when performing queries involving table aliases
  and lookups on a ``GeneratedField`` of the aliased table (:ticket:`35344`).
```
